plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/utils/get-product-types/index.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { getSetting } from '@woocommerce/settings';
+
+export type ProductTypeProps = {
```

Comment:
We now have two `ProductTypeProps` exported from two different files. Could we consolidate this into one?

Reply:
let me address this mess